docs/analytics/jenkins_jobs.md

```diff
@@ -98,6 +108,42 @@ To create job, authenticate to Jenkins, than go to main page.
         * `Send separate e-mails to individuals who broke the build`: `no`
 * Finally, click `Save`
 
+Clone existing Jenkins job
+--------------------------
+
+Once you have a [Jenkins job created](#create-new-jenkins-job), you can clone it into a new job and modify it.
+
+To do this:
+
+* Click "New Item" in the left sidebar.
+* Set the task name to be the insights domain, followed by a recognizable name that uniquely identifies the task (e.g. "insights.example.com Module Engagement"). Having the insights domain prepended to the task name will help recipients of the error emails to quickly identify the source analytics instance of any future task failure alerts.
+* Select "Copy from existing item" and select the existing task to clone from.
+* Click "Ok" to create the cloned item.
+* Modify the settings that differ (e.g. name, description, schedule crontab, command), and "Save".
+
+Import Jenkins job config.xml
+-----------------------------
+
+To perform this step reliably, you must be running with the same (or similar) version of Jenkins that the `config.xml`
+file was created with, otherwise strange problems may arise (cf https://stackoverflow.com/q/8424228/4302112).
+
+For example, to get a job's `config.xml`:
+
+```bash
+curl -X GET \
+     -o config.xml \
```

Comment:
`-O` will automatically output with the same name, but this might be better if people want to change the output name